core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -324,14 +330,13 @@ private[spark] class Executor(
     val classUri = conf.get("spark.repl.class.uri", null)
     if (classUri != null) {
       logInfo("Using REPL class URI: " + classUri)
-      val userClassPathFirst: java.lang.Boolean =
-        conf.getBoolean("spark.executor.userClassPathFirst", false)
       try {
+        val _userClassPathFirst: java.lang.Boolean = userClassPathFirst
```

Comment:
Maybe not, I'm just preserving what was already there before. This is later used in some reflection code so I think it would be safest to leave it as is.